src/test/java/hw3/shape/Rectangular.java

```diff
@@ -0,0 +1,20 @@
+package hw3.shape;
+
+public abstract class Rectangular implements Shape {
+
+    private int a;
```

Comment:
Rectangular by default should have both width and length and be overridden by Square (just swap logic you've implemented between Square and Rectangle)

Reply:
Done